crates/cargo-winrt/src/main.rs

```diff
@@ -417,14 +417,24 @@ impl DependencyDescriptor {
         }
     }
 
+    fn version(&self) -> &str {
+        match self {
+            DependencyDescriptor::NugetOrg { version, .. } => version,
+            DependencyDescriptor::Url { .. } => "unknown",
+            DependencyDescriptor::Local { .. } => "unknown",
```

Comment:
Wouldn't it be better to make these last two cases into a catch all?
```rust
_ => "unknown",
```

Reply:
I imagine we'll add more varieties and I don't want to miss changing this code when we add them

Reply:
Understood.